### ISSUE TITLE
Post preview tag link has double forward slash on multilanguage website

### ISSUE THREAD

**Issue by dovidio** [halogenica/beautifulhugo#349]
Hi, first of all thanks for this wonderful theme!

I have a multilanguage website, with some default language, something like this
```
baseURL = "/"
defaultContentLanguage = "en"
```
When I'm on the page listing post previews, all the posts with tags have a wrong link with double slash `//tags/:tagname`  which results in a 404 when clicked.
I fixed it on my website in this [commit](https://github.com/dovidio/personalwebsite/commit/34762e94c29fd2c26c16c45f8ae2de21bdf9b46d)
I'm not sure if this would break other configurations. I can create a PR if needed

**Comment by jvincentnz** [comment]
@dovidio, encountered this issue and used your [fix](https://github.com/dovidio/personalwebsite/commit/34762e94c29fd2c26c16c45f8ae2de21bdf9b46d) for `layouts/partials/post_preview.html`. I also needed to apply your fix to `layouts/_default/single.htm` to resolve all occurrences of the tag link having double forward slashes on my website.

See my PR #354 for the required changes.

**Comment by dovidio** [comment]
@jvincentnz You're right, I have overlooked that. Hopefully you're changed will get merged soon 🤞

**Comment by marcosimbuerger** [comment]
@halogenica can we merge PR https://github.com/halogenica/beautifulhugo/pull/354? Thank you!
